server/endpoints_dbg.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO cleanup this file to contain only the debug endpoints
+
+package server
+
+import (
+	ira_server "github.com/RedHatInsights/insights-results-aggregator/server"
+	"github.com/gorilla/mux"
+	"net/http"
+)
+
+const (
+	// DbgOrganizationsEndpoint returns all organizations. DEBUG only
+	DbgOrganizationsEndpoint = ira_server.OrganizationsEndpoint
+	// DbgDeleteOrganizationsEndpoint deletes all {organizations}(comma separated array). DEBUG only
+	DbgDeleteOrganizationsEndpoint = ira_server.DeleteOrganizationsEndpoint
+	// DbgDeleteClustersEndpoint deletes all {clusters}(comma separated array). DEBUG only
+	DbgDeleteClustersEndpoint = ira_server.DeleteClustersEndpoint
+	// DbgGetVoteOnRuleEndpoint is an endpoint to get vote on rule. DEBUG only
+	DbgGetVoteOnRuleEndpoint = "clusters/{cluster}/rules/{rule_id}/error_key/{error_key}/get_vote"
+)
+
+// adddbgEndpointsToRouter adds API dbg specific endpoints to the router
+func (server *HTTPServer) adddbgEndpointsToRouter(router *mux.Router) {
+	// It is possible to use special REST API endpoints in debug mode
+	if server.Config.Debug {
```

Comment:
just nitpick, but I would move this condition to the [addEndpointsToRouter](https://github.com/RedHatInsights/insights-results-smart-proxy/pull/607/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R153) since it encapsulates it encapsulates the whole body of the function here